BitwardenShared/UI/Auth/AuthCoordinator.swift

```diff
@@ -571,8 +571,13 @@ final class AuthCoordinator: NSObject, // swiftlint:disable:this type_body_lengt
         let store = Store(processor: processor)
         let view = MasterPasswordGeneratorView(store: store)
         let viewController = UIHostingController(rootView: view)
-        let navigationController = UINavigationController(rootViewController: viewController)
-        stackNavigator?.present(navigationController)
+
+        let topmostViewController = stackNavigator?.rootViewController?.topmostViewController()
+        topmostViewController?.navigationItem.backButtonTitle = Localizations.back
```

Comment:
🤔 I think for now this is fine but wanted to call out that I've had issues in the past where the `backButtonTitle` would  remain for the `topmostViewController` so if we use that controller to show a different screen will use that too. However, it doesn't seem to be the case here so I'll approve ✅